### ISSUE TITLE
Warning

### ISSUE THREAD

**Issue by shahram8708** [shahram8708/AI-ChatBot#1]
WARNING:tensorflow:From C:\Users\shobh\AppData\Local\Programs\Python\Python312\Lib\site-packages\tf_keras\src\losses.py:2976: The name tf.losses.sparse_softmax_cross_entropy is deprecated. Please use tf.compat.v1.losses.sparse_softmax_cross_entropy instead.

All model checkpoint layers were used when initializing TFGPT2LMHeadModel.

All the layers of TFGPT2LMHeadModel were initialized from the model checkpoint at microsoft/DialoGPT-medium.
If your task is similar to the task the model of the checkpoint was trained on, you can already use TFGPT2LMHeadModel for predictions without further training.   ye aa rha hai warning ismain ye wala use karke do jisse nhi aayga warning achhe se complete code likh kr do tf.compat.v1.losses.sparse_softmax_cross_entropy

**Comment by parth-patel-25** [comment]
[New Changes Files.zip](https://github.com/shahram8708/AI-ChatBot/files/15215280/New.Changes.Files.zip)

Hello Ram, I've gone through the official TensorFlow documentation, and it appears they haven't provided an official solution for the warning message yet. They mentioned they're actively working on it, and the warning you're seeing indicates that some modules are deprecated in the new APIs. This warning shouldn't affect your code's functionality, so you can safely ignore it.

Regarding the error you encountered, it seems you were trying to access tokenizer.eos_token_id[0], but tokenizer.eos_token_id is not a list object; it's an integer. Therefore, you can't access it with an index. I've fixed this issue.

Additionally, there were some front-end bugs causing the user's message to appear alongside the bot's response. I've addressed these issues as well.

You can find the Solution in the attached zip file.

**Comment by shahram8708** [comment]
Is it working properly now??
Is reponse coming from Chatbot??

**Comment by parth-patel-25** [comment]
Yeah, It is working fine responses are coming from the Bot just you have to wait for few seconds..!
